Northstar.CustomServers/mod/scripts/vscripts/weapons/_vortex.nut

```diff
@@ -478,7 +490,8 @@ function Vortex_HandleElectricDamage( entity ent, entity attacker, damage, entit
 bool function TryVortexAbsorb( entity vortexSphere, entity attacker, vector origin, int damageSourceID, entity weapon, string weaponName, string impactType, entity projectile = null, damageType = null, reflect = false )
 {
 	if ( weaponName in VortexIgnoreClassnames )
```

Comment:
Just do both on the same line
```suggestion
	if ( weaponName in VortexIgnoreClassnames && VortexIgnoreClassnames[weaponName] )
		return false
```

Reply:
I may be wrong but would that not cause an index error if classname is not in the table?

Reply:
It won't. Compilers are smart enough to immediately resolve &&s as false if any condition fails. It checks them left to right, so if the name's not in the table, it will fail the first condition and return false before it can reach the check it would error on.

Reply:
Huh, good to know, thanks!